### PR TITLE
increase timeout on tar extraction and make error clearer

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/source-to-image/issues/929

### CODE REVIEW

**Comment by bparees** [comment]
[merge]

File: pkg/tar/tar.go [hunk 1]
```diff
@@ -395,7 +395,7 @@ func (t *stiTar) ExtractTarStreamFromTarReader(dir string, tarReader Reader, log
 	})
 
 	if err != nil {
-		glog.Error("Error extracting tar stream")
+		glog.Error("Error extracting tar stream: %v", err)
```
Comment (asottile) [inline pkg/tar/tar.go]: the %v doesn't appear to do what you intend (note: I modified the timeout to hit this case in my clone)?

```
ERROR: Error extracting tar stream: %vfunction timed out after 30s
```

Comment (bparees) [reply pkg/tar/tar.go]: https://github.com/openshift/source-to-image/pull/932

Comment (asottile) [reply pkg/tar/tar.go]: dangit lol https://github.com/openshift/source-to-image/pull/933

**Pull request merged**